apps/beeai-server/src/beeai_server/application.py

```diff
@@ -36,6 +40,8 @@
 from beeai_server.telemetry import INSTRUMENTATION_NAME, shutdown_telemetry
 
 logger = logging.getLogger(__name__)
+SESSION_KEY = secrets.token_hex(16)
+middleware = [Middleware(SessionMiddleware, secret_key=SESSION_KEY, https_only=True, session_cookie="session")]
```

Comment:
Why do we need sessions in addition to access token?
This is not a distributed implementation (won't work with multiple replicas)